src/hotspot/share/runtime/deoptimization.cpp

```diff
@@ -934,113 +936,90 @@ JRT_LEAF(BasicType, Deoptimization::unpack_frames(JavaThread* thread, int exec_m
-            /* x86 */
-            (iframe->interpreter_frame_expression_stack_size() == mask.expression_stack_size() + callee_max_locals) ||
-            (try_next_mask &&
-             (iframe->interpreter_frame_expression_stack_size() == (next_mask_expression_stack_size -
-                                                                    top_frame_expression_stack_adjustment))) ||
-            (is_top_frame && (exec_mode == Unpack_exception) && iframe->interpreter_frame_expression_stack_size() == 0) ||
-            (is_top_frame && (exec_mode == Unpack_uncommon_trap || exec_mode == Unpack_reexecute || el->should_reexecute()) &&
-             (iframe->interpreter_frame_expression_stack_size() == mask.expression_stack_size() + cur_invoke_parameter_size))
-            )) {
-        {
-          // Print out some information that will help us debug the problem
-          tty->print_cr("Wrong number of expression stack elements during deoptimization");
-          tty->print_cr("  Error occurred while verifying frame %d (0..%d, 0 is topmost)", i, cur_array->frames() - 1);
-          tty->print_cr("  Current code %s", Bytecodes::name(cur_code));
-          if (try_next_mask) {
-            tty->print_cr("  Next code %s", Bytecodes::name(next_code));
-          }
-          tty->print_cr("  Fabricated interpreter frame had %d expression stack elements",
-                        iframe->interpreter_frame_expression_stack_size());
-          tty->print_cr("  Interpreter oop map had %d expression stack elements", mask.expression_stack_size());
-          tty->print_cr("  try_next_mask = %d", try_next_mask);
-          tty->print_cr("  next_mask_expression_stack_size = %d", next_mask_expression_stack_size);
-          tty->print_cr("  callee_size_of_parameters = %d", callee_size_of_parameters);
-          tty->print_cr("  callee_max_locals = %d", callee_max_locals);
-          tty->print_cr("  top_frame_expression_stack_adjustment = %d", top_frame_expression_stack_adjustment);
-          tty->print_cr("  exec_mode = %d", exec_mode);
-          tty->print_cr("  cur_invoke_parameter_size = %d", cur_invoke_parameter_size);
-          tty->print_cr("  Thread = " INTPTR_FORMAT ", thread ID = %d", p2i(thread), thread->osthread()->thread_id());
-          tty->print_cr("  Interpreted frames:");
-          for (int k = 0; k < cur_array->frames(); k++) {
-            vframeArrayElement* el = cur_array->element(k);
-            tty->print_cr("    %s (bci %d)", el->method()->name_and_sig_as_C_string(), el->bci());
-          }
-          cur_array->print_on_2(tty);
+      int iframe_expr_ssize = iframe->interpreter_frame_expression_stack_size();
+      int map_expr_invoke_ssize = mask.expression_stack_size() + cur_invoke_parameter_size;
+      int expr_ssize_before = iframe_expr_ssize + (is_top_frame ? top_frame_expression_stack_adjustment : 0);
+      int map_expr_callee_ssize = mask.expression_stack_size() + callee_size_of_parameters;
```

Comment:
`map` in these names might be more clearly `oopmap`.

Reply:
Done.